### PR TITLE
Added SCSS/SASS support, fixed weird webpack file inclusion

### PR DESCRIPTION
**Pull request by FishingHacks**

### CODE REVIEW

**Comment by FishingHacks** [comment]
why did you have the sass-loader and sass package installed?

File: server/index.ts [hunk 1]
```diff
@@ -2,42 +2,92 @@ import express from 'express';
...
+    }
+    return next();
+});
 app.use(express.static('public'));
 
-app.get('/', (req, res) => {
-	res.sendFile(path.join(__dirname, '../client/build/index.html'));
+// API
+
+app.get('/api/version', (req, res) => {
```
Comment (RedCrafter07) [inline server/index.ts:71]: Well made, although I think the version thing is not really needed

**Review by RedCrafter07 — CHANGES_REQUESTED** [review]
Very good. Thanks for the PR! Although I wouldn't put the version thing in by default. Do you see any necessity in that? I don't see APIs which have a version endpoint often.

**Comment by FishingHacks** [comment]
While yes, there's not much use to that, i have implemented it to show off, how you can use the api. I believe that often-used features have to be given as an example in an pre-configured project. The /version endpoint was a way to show the user how to use routing, the query and .send(...)/.json(...)

**Comment by RedCrafter07** [comment]
Hmm, right

**Review by RedCrafter07 — APPROVED** [review]

**Pull request merged**